product/reports/310_Migration Readiness - Providers/010_VMware Environment Summary.yaml

```diff
@@ -0,0 +1,84 @@
+---
+title: VMware Environment Summary
+rpt_type: Custom
+priority: 10
+db: Host
+cols:
```

Comment:
@martinpovolny are we creating reports with `cols` and `include` or are we starting to do reports without these optional columns?

Reply:
@kbrock : shall we modify the report editor to skip these fields?

Reply:
@martinpovolny nah - probably too much work.

Keep these columns in.